.eslintrc.json

```diff
@@ -28,6 +28,7 @@
         "react/jsx-props-no-spreading":["warn"],
         "no-param-reassign":["off"],
         "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-        "testing-library/prefer-screen-queries":["off"]
+        "testing-library/prefer-screen-queries":["off"],
+        "global-require":"off"
```

Comment:
Okay for now, let us use the exception but let's not add it globally. Can you please delete this one and add the inline exception like :`// eslint-disable-next-line global-require` to the related line ?